docs/contributing.md

```diff
@@ -7,275 +7,256 @@ layout: single
+editing the existing content to be more accessible, or creating new content such as tutorials, FAQs, etc.
 
 {{% note %}}
-A great way to understand Poetry's design and how it all fits together, is to add FAQ entries for commonly
-asked questions. Poetry members usually mark issues with [candidate/faq](https://github.com/python-poetry/poetry/issues?q=is%3Aissue+label%3Acandidate%2Ffaq+) to indicate that the issue either contains a response
-that explains how something works or might benefit from an entry in the FAQ.
+Github [Discussions](https://github.com/python-poetry/poetry/discussions) and the
+[kind/question label](https://github.com/python-poetry/poetry/labels/kind/question) are excellent sources for FAQ
+candidates.
 {{% /note %}}
 
-Issues pertaining to the documentation are usually marked with the [Documentation](https://github.com/python-poetry/poetry/labels/Documentation) label.
+Issues pertaining to the documentation are usually marked with the [area/docs label], which will also trigger a preview
+of the changes as rendered by this website.
 
-### Contributing to code
+### Code contributions
 
 #### Picking an issue
 
 {{% note %}}
-If you are a first time contributor, and are looking for an issue to take on, you might want to look for [Good First Issue](https://github.com/python-poetry/poetry/issues?q=is%3Aopen+is%3Aissue+label%3A%22Good+First+Issue%22)
-labelled issues. We do our best to label such issues, however we might fall behind at times. So, ask us.
+If you are a first time contributor, and are looking for an issue to take on, you might want to look for
+at the [contributing page](https://github.com/python-poetry/poetry/contribute) for candidates. We do our best to curate
+good issues for first-time contributors there, but do fall behind -- so if you don't see anything good, feel free to
+ask.
 {{% /note %}}
 
-If you would like to take on an issue, feel free to comment on the issue tagging `@python-poetry/triage`. We are more than happy to discuss solutions on the issue. If you would like help with navigating
-the code base, join us on our [Discord Server](https://discordapp.com/invite/awxPgve).
+If you would like to take on an issue, feel free to comment on the issue tagging `@python-poetry/triage`.
+We are more than happy to discuss solutions on the issue. If you would like help with navigating the code base, are
+looking for something to work on, or want feedback on a design or change, join us on our [Discord server][Discord] or
+start a [Discussion][Discussions].
 
 #### Local development
 
-You will need Poetry to start contributing on the Poetry codebase. Refer to the [documentation](https://python-poetry.org/docs/#introduction) to start using Poetry.
+Poetry is developed using Poetry. Refer to the [documentation] to install Poetry in your local environment.
 
 {{% note %}}
-Local development of Poetry requires Python 3.8 or newer.
+Poetry's development toolchain requires Python 3.8 or newer.
```

Comment:
```suggestion
Poetry's development toolchain requires Python 3.7 or newer.
```

Reply:
Ah, blaming led me to https://github.com/snok/flake8-type-checking, so now I see why it's 3.8+. My bad.